packages/SketchMorph2-Core.package/M2Backend.class/instance/reloadCurrentPen.st

```diff
@@ -0,0 +1,5 @@
+defaults
+reloadCurrentPen
+	self pen: self defaultPen.
+	self perform: self currentTool.
+	self penColor: self currentColor
```

Comment:
```suggestion
	self pen: self defaultPen.
	self penColor: self currentColor.
	self perform: self currentTool
```

Is sequence makes more sense to me, for example when the `currentTool` is the eraser.